### PR TITLE
[ADD] module to set version on partner from stock picking validation

### PR DESCRIPTION
**Pull request by bguillot**
Hi

This module depends on the module of this PR : OCA/partner-contact#669

It sets a version on addresses of the stock picking when the stock picking is validated.

### CODE REVIEW

File: stock_partner_version/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (rousseldenis) [inline stock_partner_version/README.rst:1]: It is recommended to use https://github.com/OCA/maintainer-tools#readme-generator to generate README from fragments

File: stock_partner_version/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion - Benoît Guillot
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Stock Partner Version',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
```
Comment (rousseldenis) [inline stock_partner_version/__manifest__.py]: Please change at least this using https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

File: stock_partner_version/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    'category': 'Stock',
+    'license': 'AGPL-3',
+    'installable': True,
+    'depends': [
+        'stock',
+        'partner_address_version',
+    ],
+    'data': [
```
Comment (rousseldenis) [inline stock_partner_version/__manifest__.py]: Please remove void entries

File: stock_partner_version/models/stock.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import models, api
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.multi
+    def action_confirm(self):
+        for picking in self:
+            picking.partner_id = picking.partner_id.get_address_version()
```
Comment (rousseldenis) [inline stock_partner_version/models/stock.py:14]: This causes errors in other tests as partner_id is not mandatory.

```suggestion
           for picking in self.filtered('partner_id'):
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

**Comment by rousseldenis** [comment]
@bguillot Do you plan to finish this ?

**Comment by bguillot** [comment]
@rousseldenis Sorry for the late answer, I made the changes.

And thank you for the review !

**Review by rousseldenis — APPROVED** [review]
Code review.

Remove change in oca_dependencies before merge

**Comment by rousseldenis** [comment]
@bguillot Do you plan to squash before merging?

**Pull request closed without merging**